test/integration/features/cmd-addons.feature

```diff
@@ -0,0 +1,158 @@
+      And stdout should match "xpaas\s*: disabled\s*P\(0\)"
+
+  Scenario: Enabling not installed add-on
+     When executing "minishift addons enable absent-addon"
+     Then stdout should contain
+      """
+      No add-on with the name absent-addon is installed.
+      """
+
+  Scenario: Enabling installed add-on
+     When executing "minishift addons enable xpaas" succeeds
+     Then stdout should contain "Add-on 'xpaas' enabled"
+      And executing "minishift addons list" succeeds
+      And stdout should match "xpaas\s*: enabled\s*P\(0\)"
+
+  Scenario: Enabling installed add-on with priority
+     When executing "minishift addons enable admin-user --priority 5" succeeds
+     Then stdout should contain "Add-on 'admin-user' enabled"
+      And executing "minishift addons list" succeeds
+      And stdout should match "admin-user\s*: enabled\s*P\(5\)"
+
+  Scenario: Starting Minishift with admin-user and xpaas add-ons enabled
+  The addons are being applied in correct order.
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+      And stdout should match
+      """
+      Applying addon 'xpaas'[\S\s]*Applying addon 'admin-user'
+      """
+      And stdout should not contain "Applying addon 'registry-route'"
+      And stdout should not contain "Applying addon 'any-uid'"
+
+  Scenario: Disabling enabled add-on
+     When executing "minishift addons disable xpaas" succeeds
+     Then stdout should contain
+      """
+      Add-on 'xpaas' disabled
+      """
+     When executing "minishift addons list" succeeds
+     Then stdout should match "xpaas\s*: disabled"
+
+  Scenario: Disabling disabled add-on
+     When executing "minishift addons disable xpaas" succeeds
+     Then stdout should contain
+      """
+      Add-on 'xpaas' disabled
+      """
+     When executing "minishift addons list" succeeds
+     Then stdout should match "xpaas\s*: disabled"
+
+  Scenario: Disabling not installed add-on
+     When executing "minishift addons disable absent-addon"
+     Then stdout should contain
+      """
+      No add-on with the name absent-addon is installed.
+      """
+
+  Scenario: Applying enabled add-on which was not applied during the startup
+    Given Minishift has state "Running"
+     When executing "minishift addons enable registry-route" succeeds
+     Then executing "minishift addons apply registry-route" succeeds
+      And stdout should contain
+      """
+      Add-on 'registry-route' created docker-registry route.
+      """
+
+  Scenario: Applying disabled add-on which was not applied during the startup
+    Given Minishift has state "Running"
+     Then executing "minishift addons apply anyuid" succeeds
+      And stdout should contain
+      """
+      Add-on 'anyuid' changed the default security context constraints to allow pods to run as any user.
+      """
+
+  Scenario: Removing applied addon
+  To be done.
```

Comment:
OOh...

Reply:
@agajdosi ,  still in progress?

Reply:
@gbraad @budhrg Oh, I have missed this, will fix it asap, sorry for that.